pkg/cmd/server/origin/openshift_apiserver.go

```diff
@@ -617,9 +616,8 @@ func apiLegacyV1(all map[string]rest.Storage) *genericapiserver.APIGroupInfo {
 		VersionedResourcesStorageMap: map[string]map[string]rest.Storage{},
 		Scheme: kapi.Scheme,
 		// version.ParameterCodec = runtime.NewParameterCodec(kapi.Scheme)
-		ParameterCodec:              kapi.ParameterCodec,
-		NegotiatedSerializer:        kapi.Codecs,
-		SubresourceGroupVersionKind: map[string]schema.GroupVersionKind{},
```

Comment:
SubresourceGroupVersionKind for scale is not needed anymore?

Reply:
seems like the test/cmd failing with error: no kind "Scale" is registered for version "extensions/v1beta1"

Reply:
see https://github.com/liggitt/origin/pull/14

Reply:
I see the errors too. I don't think manual registration is the answer... trying to track down what changed

Reply:
I've tested and verified this approach works: https://github.com/liggitt/origin/pull/15